packages/purpose-process/src/services/purposeService.ts

```diff
@@ -556,6 +561,60 @@ export function purposeServiceBuilder(
         totalCount: purposesList.totalCount,
       };
     },
+    async createPurpose(
+      purposeSeed: ApiPurposeSeed,
+      organizationId: TenantId,
+      correlationId: string
+    ): Promise<{ purpose: Purpose; isRiskAnalysisValid: boolean }> {
+      logger.info(
+        `Creating Purpose for EService ${purposeSeed.eserviceId} and Consumer ${purposeSeed.consumerId}`
+      );
+      const eserviceId = unsafeBrandId<EServiceId>(purposeSeed.eserviceId);
+      const consumerId = unsafeBrandId<TenantId>(purposeSeed.consumerId);
+      assertOrganizationIsAConsumer(organizationId, consumerId);
+
+      assertConsistentFreeOfCharge(
+        purposeSeed.isFreeOfCharge,
+        purposeSeed.freeOfChargeReason
+      );
+
+      const tenant = await retrieveTenant(organizationId, readModelService);
+
+      assertTenantKindExists(tenant);
+
+      const validatedFormSeed = validateAndTransformRiskAnalysis(
+        purposeSeed.riskAnalysisForm,
+        tenant.kind
+      );
+
+      const purposeWithSameName = await readModelService.getSpecificPurpose(
+        eserviceId,
+        consumerId,
+        purposeSeed.title
+      );
+
+      if (purposeWithSameName) {
+        throw duplicatedPurposeName(purposeSeed.title);
+      }
+
+      const purpose: Purpose = {
+        title: purposeSeed.title,
+        id: generateId(),
+        createdAt: new Date(),
+        eserviceId,
+        consumerId,
+        description: purposeSeed.description,
+        versions: [],
```

Comment:
This operation should at the same time create the first `draft` version (with the `dailyCalls` provided in `purposeSeed`).